backend/test/observer.test.js

```diff
@@ -32,4 +34,13 @@ describe('deal-observer-backend', () => {
       await observeBuiltinActorEvents(pgPool, providerMock)
     })
   })
+
+  describe('Transformer', () => {
+    it('transforms a claim event payload to a typed object', async () => {
+      const transformer = await (new Transformer().build())
+      const transformedClaimEvent = transformer.transform('ClaimEvent', claimTestEvent)
+      assert(transformedClaimEvent !== undefined, 'transformedClaimEvent is undefined')
+      assert.deepStrictEqual(transformedClaimEvent, claimTestEvent)
```

Comment:
I find this super surprising. If the transformed event is the same object as the original event, then what's the point of the transformation step?

We are missing key information here: what is the payload returned by the JSON-RPC API?